docs/src/continous-integration.rst

```diff
@@ -0,0 +1,94 @@
+*********************************
+Continuous Integration (CircleCI)
+*********************************
+
+.. admonition:: What is continuous integration?
+
+   Continuous integration (CI) is a software development strategy that increases the speed of development while ensuring
+   the quality of the code that teams deploy. Developers continually commit code in small increments (at least daily, or
+   even several times a day), which is then automatically built and tested before it is merged with the shared repository.
+
+   Source: `CircleCI documentation <https://circleci.com/continuous-integration/>`__
+
+Configuration
+=============
+
+Our CircleCI configuration resides in :github-source:`.circleci/config.yml`.
+It contains all the jobs listed below.
+See `Configuring CircleCI <https://circleci.com/docs/2.0/configuration-reference/>`__ for a full reference.
+
+Workflow ``develop``
```

Comment:
Could you also add documentation about the `main` and `deploy` workflows?
In general, you can copy-paste the explanations from the integreat-cms documentation, just make sure to adapt them where necessary (e.g. instead of pipenv install, we do a normal pip install etc.)